### DISCUSSION TITLE
Does sqlalchemy/alembic support databricks SQL?

### DISCUSSION THREAD

**Discussion by shibukv** [sqlalchemy/alembic#995] [Usage Questions]
We have a python app which connects to databricks SQL. Looking for a schema migration tool. Does sqlalchemy/alembic support databricks SQL?

**Comment by CaselIT** [comment]
Hi,

I think you would need to look for an external driver, since there is no internal support for it.
If it's a variation on another sql db you may be able to use that, like there are various postgresql bases db, but for better compatibility a dedicated dialect is always suggested
